jakarta.jdt/org.eclipse.lsp4jakarta.jdt.test/src/main/java/org/eclipse/lsp4jakarta/jdt/websocket/JakartaWebSocketTest.java

```diff
@@ -57,7 +65,13 @@ public void addPathParamsAnnotation() throws Exception {
                 DiagnosticSeverity.Error, "jakarta-websocket", "AddPathParamsAnnotation"
         );
 
-        assertJavaDiagnostics(diagnosticsParams, JDT_UTILS, d1, d2, d3);        
+        assertJavaDiagnostics(diagnosticsParams, JDT_UTILS, d1, d2, d3);
+        
+        // Expected code actions
+        JakartaJavaCodeActionParams codeActionsParams = createCodeActionParams(uri, d1);
+        TextEdit te = te(4, 32, 17, 40, "\nimport jakarta.websocket.server.PathParam;\nimport jakarta.websocket.server.ServerEndpoint;\nimport jakarta.websocket.Session;\n\n/**\n * Expected Diagnostics are related to validating that the parameters have the \n * valid annotation @PathParam (code: AddPathParamsAnnotation)\n * See issue #247 (onOpen) and #248 (onClose)\n */\n@ServerEndpoint(value = \"/infos\")\npublic class AnnotationTest {\n    // @PathParam missing annotation for \"String missingAnnotation\"\n    @OnOpen\n    public void OnOpen(Session session, @PathParam(value = \"\") ");
```

Comment:
Would it be possible to move the long string to its own variable? It would make the code more readable.

Reply:
I implemented this suggestion in my new commit! Thanks!